utils/_features.py

```diff
@@ -2426,3 +2426,13 @@ def serverless_span_pointers(test_object):
         """
         pytest.mark.features(feature_id=328)(test_object)
         return test_object
+    
+    @staticmethod
+    def aws_api_gateway_inferred_span_creation(test_object):
+        """
+        AWS Api Gateway: Tests that API Gateway inferred span creation works as expected
+
+        https://feature-parity.us1.prod.dog/#/?feature=329
+        """
+        pytest.mark.features(feature_id=329)(test_object)
```

Comment:
Checking this on FPD I can see the feature 329 is "Attach stack trace to IAST vulnerabilities"

Reply:
fixed